src/libraries/Microsoft.Extensions.DependencyInjection/src/ServiceProviderOptions.cs

```diff
@@ -19,8 +19,11 @@ public class ServiceProviderOptions
         public bool ValidateScopes { get; set; }
 
         /// <summary>
-        /// Gets or sets a value that indicates whether validation is performed to ensure all services can be created when <see cref="M:BuildServiceProvider(IServiceCollection,ServiceProviderOptions)" /> is called.
+        /// Gets or sets a value that indicates whether validation is performed to ensure all services can be created when <see cref="BuildServiceProvider(IServiceCollection, ServiceProviderOptions)" /> is called.
         /// </summary>
+        /// <value>
+        /// <see langword="true" /> if validation is performed to ensure all services can be created during a <see cref="BuildServiceProvider(IServiceCollection, ServiceProviderOptions)" /> call; otherwise, <see langword="false" />. The default is the value of <see cref="Microsoft.Extensions.Hosting.HostingEnvironmentExtensions.IsDevelopment(Microsoft.Extensions.Hosting.IHostingEnvironment)" />.
```

Comment:
```suggestion
        /// <see langword="true" /> if validation is performed to ensure all services can be created during a <see cref="ServiceCollectionContainerBuilderExtensions.BuildServiceProvider(IServiceCollection, ServiceProviderOptions)" /> call; otherwise, <see langword="false" />. The default is the value of <see cref="Microsoft.Extensions.Hosting.HostingEnvironmentExtensions.IsDevelopment(Microsoft.Extensions.Hosting.IHostingEnvironment)" />.
```